src/SerialLoops/Controls/ItemListPanel.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Eto.Drawing;
+using Eto.Forms;
+using HaruhiChokuretsuLib.Util;
+using SerialLoops.Lib;
+using SerialLoops.Lib.Items;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SerialLoops.Controls
+{
+    public abstract class ItemListPanel : Scrollable
+    {
+        private List<ItemDescription> _items;
+        private bool _expandItems { get; set; }
```

Comment:
this is probably my fault but private members shouldn't have accessors lol oops
```suggestion
        private bool _expandItems;
```